etc/setup/requirements.txt

```diff
@@ -0,0 +1,159 @@
+absl-py==0.10.0
```

Comment:
Why is the `requirements.txt` file present in two locations, namely, `SMARTS/etc/setup/` and `SMARTS/` ?

Reply:
I was checking whether moving the requirements file would fix the CI test failure. I removed the requirements.txt from etc/setup.